include/fmt/format.h

```diff
@@ -99,6 +99,13 @@
 #  define FMT_VISIBILITY(value)
 #endif
 
+#if !defined(FMT_HEADER_ONLY) && !defined(_WIN32) && \
+    (defined(FMT_LIB_EXPORT) || defined(FMT_SHARED))
+#  define FMT_INLINE_API FMT_VISIBILITY("default")
+#else
+#  define FMT_INLINE_API
+#endif
```

Comment:
I don't think we should introduce a new macro. Let's update `FMT_VISIBILITY` instead since we shouldn't be changing visibility in general when not compiling a shared library.

Reply:
@vitaut 
FMT_VISIBILITY is used unconditionally:
https://github.com/fmtlib/fmt/blob/a4b7b24b7b20d5462565e9cadbdfc7f101a4b843/include/fmt/format.h#L1885

Therefore, we need two macros.

Reply:
OK, we can simplify this later.